README.md

```diff
@@ -173,6 +173,7 @@ flags
   -e character to use for plot error line when value exceeds hardmax (default: e)
   -s initial scale of the plot (can go above if data input has larger value)
   -m maximum value, if exceeded draws error line (see -e), plot scale is fixed
+  -M minimum value, if not exceeded draws error line (see -e), plot scale is fixed
```

Comment:
I think this it little bit unfortunate wording. Doesn't really explain whats going on there. I think it should say something about if value is lower than minimum then.... It should also say that the min or lower scale is fixed, as max is not. This also begs to fix line above to make it more specific about upper/max scale.

Reply:
Also upon testing I'm not convinced that printing error line throughout the whole vertical scale is appropriate in this case. This makes sense for > max but not for < min. I think maybe print 'm' or just "min", or something like "v" or down arrow on the bottom of the scale would be better?